css-basics/css-basics_pokemon-page/css/styles.css

```diff
@@ -21,6 +22,14 @@
 /* Task 3: To set the line height to 1.5 is good for texts, because of accessibility concerns, but for the headings it seems too much. Let's change this by giving our headings a line height of 1.2: */
 
 /* your solution */
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  line-height: 1.5;
```

Comment:
I believed the exercise asks to update to line height of 1.2. You might want to update this.